project/AppDependencies.scala

```diff
@@ -20,7 +20,7 @@ import sbt.*
 
 object AppDependencies {
 
-  val bootstrapPlayVersion = "8.1.0"
+  val bootstrapPlayVersion = "8.6.0"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
```

Comment:
There is an updated version of cats-core -> `2.13`, some of these deps can be removed. Can refer to Scala 3 spike for this